lib/matplotlib/figure.py

```diff
@@ -883,8 +929,7 @@ def set_size_inches(self, w, h=None, forward=True):
 
         # the width and height have been passed in as a tuple to the first
         # argument, so unpack them
-        if h is None:
```

Comment:
I think we still need this special casing here.

Reply:
Sorry, bad cut and paste job...